dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/RegistrationEndpointServices.java

```diff
@@ -680,14 +620,10 @@ private OidcOAuth20Client validateClientIdExists(String clientId, OidcOAuth20Cli
     private String validateRequestContainsClientId(HttpServletRequest request) throws OidcServerException {
         String clientId = extractClientId(request.getPathInfo());
         if (OidcOAuth20Util.isNullEmpty(clientId)) {
-            String browserErrorMsg = Tr.formatMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_MISSING_CLIENTID", new Object[] { request.getMethod(), OAuth20Constants.CLIENT_ID });
-            String serverErrorMsg = TraceNLS.getFormattedMessage(RegistrationEndpointServices.class,
-                    MESSAGE_BUNDLE,
-                    "OAUTH_CLIENT_REGISTRATION_MISSING_CLIENTID",
-                    new Object[] { request.getMethod(), OAuth20Constants.CLIENT_ID },
-                    "CWWKS1426E: The {0} operation failed as the request did not contain the {1} parameter.");
-            Tr.error(tc, serverErrorMsg);
-            throw new OidcServerException(browserErrorMsg, OIDCConstants.ERROR_INVALID_REQUEST, HttpServletResponse.SC_BAD_REQUEST);
+            BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_MISSING_CLIENT_ID", new Object[] { clientId });
```

Comment:
The message key in the update (`OAUTH_CLIENT_REGISTRATION_MISSING_CLIENT_ID`) doesn't match the original key (`OAUTH_CLIENT_REGISTRATION_MISSING_CLIENTID`).